src/main/java/com/fewlaps/quitnowcache/QNCache.java

```diff
@@ -195,4 +198,29 @@ private String getEffectiveKey(String key) {
         }
         return key;
     }
+    
+    /**
+     * Find 
+     */
+    List<String> findStartsWith(String someCuquiWord) {
+    	final String key = getEffectiveKey(someCuquiWord);
+
+    	return Collections.list(cache.keys())
+    			.stream()
+    			.filter(s -> s.startsWith(key))
+    			.collect(Collectors.toList());
+    }
+    
+    /**
+     * Finding coincidences
+     */
+    List<String> findStartsWithIfAlive(String someCuquiWord) {
```

Comment:
- > As the method to get elements from the cache is called get, what do you think about naming this method getItemsStartingWith? It will be more discoverable when someone types ge and the IDE suggests get and this new method.

This method don't get elements and only return keys so name is not bad!

- > Probably there are people that doesn't know what cuqui means :·D What's the name of the Java startsWith method? Probably it will be a better one.

Damn!,... "Cuqui" isn't universal? :(